source/element-kinds.md

```diff
@@ -407,9 +406,9 @@ For other purposes, for example, to mark reference points, [Marker](#s.marker) e
 (s:octupole)=
 ## Octupole Element
 
-Octupole element.
-
-Under Construction...
+An `octupole` is an element whose major field has a cubic field dependence with transverse offset.
+Both electric and magnetic fields can be defined and additional multipole contributions are allowed.
+In terms of functionality, an `octupole` is equivalent to a [`Multipole`](#s:multipole) element.
```

Comment:
What does "major field" mean in this context? Does it imply any constraint on the field components that should be set up or can an octupole have, say, `KnN` components of any order `N`?

In other words, should/could we validate somehow whether an octupole is indeed an octupole? 

I would ask the same questions for sextupole, etc.